must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Make a globle variable for namespace
+INSTALL_NAMESPACE=openshift-storage
+
+# Add general resources to list if necessary
+
+# Resource List
+resources=()
+# collect storagecluster resources
+
+# TODO: Re enable the collection of storagecluster via inspect command
+# resources+=(storageclusters)
+
+# collect OB/OBC resoureces
+resources+=(objectbucketclaims)
+resources+=(objectbuckets)
+
+# collection path for OC commands
+mkdir -p ${BASE_COLLECTION_PATH}/oc_output/
+
+# Command List
+commands_get=()
+
+# collect oc output of OC get commands
+commands_get+=("pods -owide")
+commands_get+=("nodes --show-labels")
+commands_get+=("subscription")
+commands_get+=("csv")
+commands_get+=("installplan")
+commands_get+=("events")
+commands_get+=("all")
+commands_get+=("role")
+commands_get+=("rolebinding")
+
+# collect oc output of OC desc commands
+commands_desc=()
+commands_desc+=("nodes")
+commands_desc+=("pods")
+commands_desc+=("storagecluster")
+
+# collect yaml output of OC commands
+oc_yamls=()
+oc_yamls+=("csv")
+oc_yamls+=("installplan")
+
+echo "collecting dump of namespace" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect ns/"${INSTALL_NAMESPACE}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+echo "collecting dump of clusterresourceversion" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+for oc_yaml in "${oc_yamls[@]}"; do
+   # shellcheck disable=SC2129
+   oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect "${oc_yaml}" -n "${INSTALL_NAMESPACE}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+done
+
+# Create the dir for oc_output
+mkdir -p ${BASE_COLLECTION_PATH}/namespaces/openshift-storage/oc_output/
```

Comment:
All instances also includes instances in file paths. :)

Reply:
fixed.

Reply:
Weird! Okay, can we find or raise an issue with the oc repo and document it here? Doesn't have to be in this PR, I won't hold it up for that.